test/Microsoft.AspNet.Mvc.IntegrationTests/TryValidateModelIntegrationTest.cs

```diff
@@ -140,7 +140,7 @@ public void TryValidateModel_CollectionsModel_ReturnsErrorsForInvalidProperties(
         {
             var controller = new TestController();
             controller.ControllerContext = new ControllerContext(operationContext.ActionContext);
-            controller.ObjectValidator = ModelBindingTestHelper.GetObjectValidator();
+            controller.ObjectValidator = ModelBindingTestHelper.GetObjectValidator(operationContext.MetadataProvider);
```

Comment:
Actually this was a change that I had to make after rebasing. This change was triggered after the commit https://github.com/aspnet/Mvc/commit/96de1dbe4b914734cfe97f271230ff6c6ddcfa11